php/templates/taxonomy-transformation-fields.php

```diff
@@ -12,7 +12,7 @@
 
 wp_add_inline_script( 'cloudinary', 'var CLD_GLOBAL_TRANSFORMATIONS = CLD_GLOBAL_TRANSFORMATIONS ? CLD_GLOBAL_TRANSFORMATIONS : {};', 'before' );
 
-$tax_slug   = filter_input( INPUT_GET, 'taxonomy', FILTER_SANITIZE_STRING );
+$tax_slug   = filter_input( INPUT_GET, 'taxonomy', FILTER_CALLBACK, array( 'options' => 'sanitize_text_field' ) );
```

Comment:
@pereirinha - Since we're doing this so often, perhaps it would be a good idea to make it a Util.
```php
	/**
	 * Get a sanitized input text field.
	 *
	 * @param string $var  The value to get.
	 * @param int    $type The type to get.
	 *
	 * @return mixed
	 */
	public static function get_sanitized_text( $var, $type = INPUT_GET ) {
		return filter_input( $type, $var, FILTER_CALLBACK, array( 'options' => 'sanitize_text_field' ) );
	}


$tax_slug = Utils::get_sanitized_text( 'taxonomy' );
```

Reply:
Good idea ;)